datadog_checks_base/tests/base/utils/db/test_util.py

```diff
@@ -112,51 +107,37 @@ class TestDBExcepption(BaseException):
                     'tables': ['datadog', 'datadog2'],
                 },
             },
-            True,
         ),
         (
-            json.dumps(
-                {
-                    'query': 'COMMIT',
-                    'metadata': {'tables_csv': '', 'commands': ['COMMIT'], 'comments': None},
-                }
-            ),
+            "{\"query\":\"COMMIT\",\"metadata\":{\"tables_csv\":\"\",\"commands\":[\"COMMIT\"],\"comments\":null}}",
```

Comment:
Why put raw json here? The previous json.dumps should have worked?

Reply:
oh heh, this was before updating to support that behavior, let me revert.

Reply:
Reverted, kept the one that tests whitespace.